[Issue] Linux Pytorch: `TestMemPool::test_snapshot_include_traces` fails to compile extension: `hipblas/hipblas.h` not found

### Description:
 The job ends with exit code 134 (Aborted) due to a missing HIPBLAS header.

### Context:
**Platform:** `Linux`
**Effected Archs:** `gfx950, gfx94x`
**Pytorch Versions:** `2.11, nightly`

### Full logs:
- https://github.com/ROCm/TheRock/actions/runs/24385542651/job/71227048052
- https://github.com/ROCm/TheRock/actions/runs/24384283164/job/71218109233
- https://github.com/ROCm/TheRock/actions/runs/24384283164/job/71218186425

### Error Log:
```
/__w/TheRock/TheRock/.venv/lib/python3.10/site-packages/torch/include/torch/csrc/api/include -isystem /__w/TheRock/TheRock/.venv/lib/python3.10/site-packages/torch/include/THH -isystem /__w/TheRock/TheRock/.venv/include -isystem /__w/_tool/Python/3.10.20/x64/include/python3.10 -fPIC -std=c++17 -c /github/home/.cache/torch_extensions/py310_rocm71360980/dummy_allocator/main_hip.cpp -o main_hip.o -D__HIP_PLATFORM_AMD__=1 -DUSE_ROCM=1 -DHIPBLAS_V2 -fPIC
2026-04-14T08:40:09.9328829Z FAILED: [code=1] main_hip.o 
2026-04-14T08:40:09.9332200Z c++ -MMD -MF main_hip.o.d -DTORCH_EXTENSION_NAME=dummy_allocator -DTORCH_API_INCLUDE_EXTENSION_H -isystem /__w/TheRock/TheRock/.venv/lib/python3.10/site-packages/torch/include -isystem /__w/TheRock/TheRock/.venv/lib/python3.10/site-packages/torch/include/torch/csrc/api/include -isystem /__w/TheRock/TheRock/.venv/lib/python3.10/site-packages/torch/include/THH -isystem /__w/TheRock/TheRock/.venv/include -isystem /__w/_tool/Python/3.10.20/x64/include/python3.10 -fPIC -std=c++17 -c /github/home/.cache/torch_extensions/py310_rocm71360980/dummy_allocator/main_hip.cpp -o main_hip.o -D__HIP_PLATFORM_AMD__=1 -DUSE_ROCM=1 -DHIPBLAS_V2 -fPIC
2026-04-14T08:40:09.9335977Z In file included from /github/home/.cache/torch_extensions/py310_rocm71360980/dummy_allocator/main_hip.cpp:5:
2026-04-14T08:40:09.9337219Z /__w/TheRock/TheRock/.venv/lib/python3.10/site-packages/torch/include/ATen/hip/Exceptions.h:5:10: fatal error: 'hipblas/hipblas.h' file not found
2026-04-14T08:40:09.9338641Z     5 | #include <hipblas/hipblas.h>
2026-04-14T08:40:09.9339017Z       |          ^~~~~~~~~~~~~~~~~~~
2026-04-14T08:40:09.9339380Z 1 error generated.
2026-04-14T08:40:09.9339725Z ninja: build stopped: subcommand failed.
2026-04-14T08:40:09.9349054Z FAILED [4.8538s] [  8%]
2026-04-14T08:40:09.9360741Z external-builds/pytorch/pytorch/test/test_cuda.py::TestMemPool::test_mempool_id PASSED [0.0008s] [  8%]
2026-04-14T08:40:09.9386312Z external-builds/pytorch/pytorch/test/test_cuda.py::TestMemPool::test_mempool_multithread PASSED [0.0021s] [  8%]
2026-04-14T08:40:09.9497318Z external-builds/pytorch/pytorch/test/test_cuda.py::TestMemPool::test_mempool_no_split FAILED [0.0107s] [  8%]
2026-04-14T08:40:09.9584112Z external-builds/pytorch/pytorch/test/test_cuda.py::TestMemPool::test_nested_mempool FAILED [0.0082s] [  8%]
2026-04-14T08:40:09.9630569Z terminate called after throwing an instance of 'c10::Error'
2026-04-14T08:40:09.9637077Z   what():  captures_underway.empty() INTERNAL ASSERT FAILED at "/__w/TheRock/TheRock/external-builds/pytorch/pytorch/c10/hip/HIPCachingAllocator.cpp":3781, please report a bug to PyTorch. 
2026-04-14T08:40:09.9638852Z Exception raised from synchronize_and_free_events at /__w/TheRock/TheRock/external-builds/pytorch/pytorch/c10/hip/HIPCachingAllocator.cpp:3781 (most recent call first):
```

--- a/external-builds/pytorch/skip_tests/pytorch_2.11.py
+++ b/external-builds/pytorch/skip_tests/pytorch_2.11.py
@@ -32,6 +32,9 @@ skip_tests = {
             "test_tensor_delete_after_allocator_delete",
             # RuntimeError: Error building extension 'dummy_allocator'
             "test_deleted_mempool_not_used_on_oom",
+            # Same hipblas.h compilation error as test_mempool_with_allocator.
+            # See https://github.com/pytorch/pytorch/pull/173330
+            "test_mempool_expandable",
             # ModuleNotFoundError: No module named 'torchvision'
             "test_resnet",
             # RuntimeError: miopenStatusUnknownError

